endogenous/scripts/end_transport.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```

Comment:
The usage of _'_ or _"_ to delimit/specify literal strings is a matter of taste in Python and its community use both (i.e. it is a matter of personal taste). However, to ease maintenance and readability, it is important to use one or the other consistently throughout the code.